src/EnergyPlus/HeatBalanceIntRadExchange.cc

```diff
@@ -667,7 +665,7 @@ namespace HeatBalanceIntRadExchange {
                               thisEnclosure.Emissivity(SurfNum),
                               state.dataSurface->Surface(thisEnclosure.SurfacePtr(SurfNum)).Sides);
                         for (int Vindex = 1; Vindex <= state.dataSurface->Surface(thisEnclosure.SurfacePtr(SurfNum)).Sides; ++Vindex) {
-                            auto &Vertex = state.dataSurface->Surface(thisEnclosure.SurfacePtr(SurfNum)).Vertex(Vindex);
+                            auto const &Vertex = state.dataSurface->Surface(thisEnclosure.SurfacePtr(SurfNum)).Vertex(Vindex);
```

Comment:
If I can tell at a glance that a `&` or `*` can be `const` I will add it. `Cpp-check` will find the rest.

Reply:
Sometimes I just add const and compile. If it's valid it will compile.